fiat-plugins/fiat-plugins.gradle

```diff
@@ -0,0 +1,25 @@
+subprojects {
+  apply plugin: 'distribution'
+
+  distributions {
+    main {
+      contents {
+        from jar.outputs
+        from {
+          configurations.runtimeClasspath.resolvedConfiguration.resolvedArtifacts.findResults { dep ->
```

Comment:
this is the main craziness that strips out any dependency that comes along with the core app from the plugin distribution (if we wanted to go nested classloaders or anything this might make it simpler)

also worth noting - I had to rename fiat-web to fiat-main because I wanted to separate the -web code from the packaging (so I could have the packaging include all these known plugins into the distribution and figure out how to hook them up with our plugin loading story) - however we have some stuff in our gradle plugins that expects the packaging stuff to live in a project named `-web` so (don't remember why)